core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -130,6 +130,11 @@ private[spark] class Executor(
   private val urlClassLoader = createClassLoader()
   private val replClassLoader = addReplClassLoaderIfNeeded(urlClassLoader)
 
+  // Load executor plugins
```

Comment:
Might be useful to have logDebug before and after these calls so if necessary we can see it trying to load plugins and then when its finished.  Might help more easily find bad plugins.  If others think its overkill I'm ok leaving it out though